src/products/src/products-service/product.go

```diff
@@ -7,18 +7,18 @@ package main
 // using omitempty as a DynamoDB optimization to create indexes
 // IMPORTANT: if you change the shape of this struct, be sure to update the retaildemostore-lambda-load-products Lambda too!
 type Product struct {
-	ID             string  `json:"id" yaml:"id"`
-	URL            string  `json:"url" yaml:"url"`
-	SK             string  `json:"sk" yaml:"sk"`
-	Name           string  `json:"name" yaml:"name"`
-	Category       string  `json:"category" yaml:"category"`
-	Style          string  `json:"style" yaml:"style"`
-	Description    string  `json:"description" yaml:"description"`
-	Price          float32 `json:"price" yaml:"price"`
-	Image          string  `json:"image" yaml:"image"`
-	Featured       string  `json:"featured,omitempty" yaml:"featured,omitempty"`
-	GenderAffinity string  `json:"gender_affinity,omitempty" yaml:"gender_affinity,omitempty"`
-	CurrentStock   int     `json:"current_stock" yaml:"current_stock"`
+	ID               string  `json:"id" yaml:"id"`
+	URL              string  `json:"url" yaml:"url"`
+	SK               string  `json:"sk" yaml:"sk"`
+	Name             string  `json:"name" yaml:"name"`
+	Category         string  `json:"category" yaml:"category"`
+	Style            string  `json:"style" yaml:"style"`
+	Description      string  `json:"description" yaml:"description"`
+	Price            float32 `json:"price" yaml:"price"`
+	Image            string  `json:"image" yaml:"image"`
+	Featured         string  `json:"featured,omitempty" yaml:"featured,omitempty"`
+	GenderAffinity   string  `json:"gender_affinity,omitempty" yaml:"gender_affinity,omitempty"`
+	CurrentStock     int     `json:"current_stock" yaml:"current_stock"`
```

Comment:
Aliases is not in the field list but is loaded by the catalog load lambda. Was it left out on purpose?

Reply:
Perhaps this dropped off while testing the product fields for the filters. An oversight, at any rate. I have added it back on.